tests/discontinuous_composition_1/screen-output

```diff
@@ -0,0 +1,456 @@
+
+*** Timestep 27:  t=0.84375 seconds
+   Solving temperature system... 4 iterations.
+   Solving C_1 system ... 3 iterations.
+   Solving C_2 system ... 4 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 30 iterations.
+
+   Postprocessing:
+     Temperature min/avg/max:   0 K, 0.4975 K, 1 K
+     Compositions min/max/mass: -0.1193/1.147/0.4583 // -0.3186/1.268/0.3959
+
+*** Timestep 28:  t=0.875 seconds
+   Solving temperature system... 4 iterations.
+   Solving C_1 system ... 3 iterations.
+   Solving C_2 system ... 4 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 30 iterations.
+
+   Postprocessing:
+     Temperature min/avg/max:   0 K, 0.4975 K, 1 K
+     Compositions min/max/mass: -0.1345/1.162/0.4583 // -0.3291/1.264/0.3959
+
+Number of active cells: 247 (on 5 levels)
+Number of degrees of freedom: 7,897 (2,114+280+1,057+2,223+2,223)
+
+*** Timestep 29:  t=0.90625 seconds
+   Solving temperature system... 4 iterations.
+   Solving C_1 system ... 3 iterations.
+   Solving C_2 system ... 4 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 30+3 iterations.
+
+   Postprocessing:
+     Temperature min/avg/max:   0 K, 0.4974 K, 1 K
+     Compositions min/max/mass: -0.1695/1.176/0.4583 // -0.3509/1.295/0.3959
+
+*** Timestep 30:  t=0.9375 seconds
+   Solving temperature system... 4 iterations.
+   Solving C_1 system ... 3 iterations.
+   Solving C_2 system ... 4 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 30+3 iterations.
+
+   Postprocessing:
+     Temperature min/avg/max:   0 K, 0.4974 K, 1 K
+     Compositions min/max/mass: -0.1883/1.183/0.4583 // -0.3627/1.323/0.3959
+
+Number of active cells: 256 (on 5 levels)
+Number of degrees of freedom: 8,164 (2,178+289+1,089+2,304+2,304)
+
+*** Timestep 31:  t=0.96875 seconds
+   Solving temperature system... 4 iterations.
+   Solving C_1 system ... 3 iterations.
+   Solving C_2 system ... 4 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 30 iterations.
+
+   Postprocessing:
+     Temperature min/avg/max:   0 K, 0.4974 K, 1 K
+     Compositions min/max/mass: -0.194/1.184/0.4583 // -0.3584/1.335/0.3959
+
+*** Timestep 32:  t=1 seconds
+   Solving temperature system... 4 iterations.
+   Solving C_1 system ... 3 iterations.
+   Solving C_2 system ... 4 iterations.
+   Rebuilding Stokes preconditioner...
+   Solving Stokes system... 30 iterations.
+
+   Postprocessing:
+     Temperature min/avg/max:   0 K, 0.4974 K, 1 K
+     Compositions min/max/mass: -0.1925/1.195/0.4583 // -0.3429/1.367/0.3959
+
+Number of active cells: 250 (on 5 levels)
+Number of degrees of freedom: 7,997 (2,142+284+1,071+2,250+2,250)
+
+Termination requested by criterion: end time
+
+
++---------------------------------------------+------------+------------+
+| Total wallclock time elapsed since start    |       105s |            |
```

Comment:
do you think that test requires 32 timesteps to be useful? 105 seconds is not exactly quick.